src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationsTest.java

```diff
@@ -90,6 +90,105 @@ void persists_and_returns_organisation_details() {
+
+        Map<String, Object> orgResponse2 =
+            professionalReferenceDataClient.retrieveAllOrganisationsWithPagination("1", "3", hmctsAdmin);
+
+        Map<String, Object> orgResponse3 =
+            professionalReferenceDataClient.retrieveAllOrganisationsWithPagination("1", "5", hmctsAdmin);
+
+        int orgResponse1Size = ((List<Organisation>) orgResponse1.get("organisations")).size();
+        int orgResponse2Size = ((List<Organisation>) orgResponse2.get("organisations")).size();
+        int orgResponse3Size = ((List<Organisation>) orgResponse3.get("organisations")).size();
+
+        assertThat(orgResponse1).containsEntry("http_status","200 OK");
+        assertThat(orgResponse2).containsEntry("http_status","200 OK");
+        assertThat(orgResponse3).containsEntry("http_status","200 OK");
+        assertThat(orgResponse1Size).isEqualTo(2);
+        assertThat(orgResponse2Size).isEqualTo(3);
+        assertThat(orgResponse3Size).isEqualTo(5);
+    }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    void persists_and_returns_organisation_with_default_pagination() {
+        String orgIdentifierResponse1 = createOrganisationRequest("PENDING");
+        String orgIdentifier2 = createAndActivateOrganisationWithGivenRequest(
+            someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+        String orgIdentifier3 = createOrganisationWithGivenRequest(
+            someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+        String orgIdentifier4 = createOrganisationWithGivenRequest(
+            someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+        String orgIdentifier5 = createOrganisationWithGivenRequest(
+            someMinimalOrganisationRequest().status("ACTIVE").sraId(randomAlphabetic(10)).build());
+        assertThat(orgIdentifierResponse1).isNotEmpty();
+        assertThat(orgIdentifier2).isNotEmpty();
+        assertThat(orgIdentifier3).isNotEmpty();
+        assertThat(orgIdentifier4).isNotEmpty();
+        assertThat(orgIdentifier5).isNotEmpty();
+
+        Map<String, Object> orgResponseWithDefaultPageSize =
+            professionalReferenceDataClient.retrieveAllOrganisationsWithPagination("1", null, hmctsAdmin);
+
+        Map<String, Object> orgResponseWithDefaultPage =
+            professionalReferenceDataClient.retrieveAllOrganisationsWithPagination(null, "2", hmctsAdmin);
+
+        int orgResponse1Size = ((List<Organisation>) orgResponseWithDefaultPageSize.get("organisations")).size();
+        int orgResponse2Size = ((List<Organisation>) orgResponseWithDefaultPage.get("organisations")).size();
+
+        assertThat(orgResponseWithDefaultPageSize).containsEntry("http_status","200 OK");
+        assertThat(orgResponseWithDefaultPage).containsEntry("http_status","200 OK");
+        assertThat(orgResponse1Size).isEqualTo(5);
+        assertThat(orgResponse2Size).isEqualTo(2);

```

Comment:
1.add assertions for order of the expected response content
2.assert expected total records in response header